### DISCUSSION TITLE
how to chang LoginPath and ExpireTimeSpan ?

### DISCUSSION THREAD

**Discussion by duanwenhao** [abpframework/abp#15846] [Q&A]
with .net core i can do it by under code
services.AddAuthentication(CookieAuthenticationDefaults.AuthenticationScheme)
                .AddCookie(options =>
                {
                    options.LoginPath = "/Login";
                    options.ExpireTimeSpan = TimeSpan.FromMinutes(15);
                    options.SlidingExpiration = true;
                });

abp's default loginpath is /Account/Login , how can i change it?

**Comment by maliming** [comment]
https://learn.microsoft.com/en-us/aspnet/core/security/authentication/identity-configuration?view=aspnetcore-7.0#cookie-settings
